scrutiny.json

```
@@ -0,0 +1,82 @@
+                    ],
+                    [
+                        "--device",
+                        "/dev:/dev"
+                    ],
+                    [
+                        "-v",
+                        "/run/udev:/run/udev:ro"
+                    ],
+                    [
+                        "--privileged",
+                        ""
+                    ]
+                ],
+                "ports" : {
+                    "8080": {
+                        "description": "Port for scrutiny's web interface and API.",
+                        "host_default": 8080,
+                        "label": "WebUI HTTP port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for Scrutiny config files. Eg: create a Share called Scrutiny-Share-Config for this purpose alone.",
+                        "label": "Config share"
+                    }
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter a valid system UserID for Scrutiny. Must have full permissions on the previous Share.",
+                        "label": "UID",
+                        "index": 1
+                    },
+                    "PGID": {
+                        "description": "Enter a valid system GroupID for Scrutiny. This group or the above UserID must have full permissions on the previous Share.",
+                        "label": "GID",
+                        "index": 2
+                    },
+                    "SCRUTINY_API_ENDPOINT": {
+                        "description": "API endpoint of the scrutiny UI. Do not change unless using as a remote collector. Default: 'http://localhost:8080'",
+                        "label": "SCRUTINY_API_ENDPOINT",
+                        "index": 3
+                    },
+                    "SCRUTINY_WEB": {
+                        "description": "Run the web service. Default: 'true'",
+                        "label": "SCRUTINY_WEB",
+                        "index": 4
+                    },
+                    "SCRUTINY_COLLECTOR": {
+                        "description": "Run the metrics collector. Default: 'true'",
+                        "label": "SCRUTINY_COLLECTOR",
+                        "index": 5
+                    }
+                }
+            }
+        },
+        "description": "<p><a href='https://github.com/AnalogJ/scrutiny'>Scrutiny</a> WebUI for smartd S.M.A.R.T monitoring. Scrutiny is a Hard Drive Health Dashboard &amp; Monitoring solution, merging manufacturer provided S.M.A.R.T metrics with real-world failure rates from Backblaze.</p><p>Based on custom docker image: <a href='https://hub.docker.com/r/linuxserver/scrutiny'>https://hub.docker.com/r/linuxserver/scrutiny</a>, available for x86-64, arm64, and armhf</p>",
```

Minor links editing:
```suggestion
        "description": "<p>WebUI for smartd S.M.A.R.T monitoring. Scrutiny is a Hard Drive Health Dashboard &amp; Monitoring solution, merging manufacturer provided S.M.A.R.T metrics with real-world failure rates from Backblaze.</p><p>Based on custom docker image: <a target='_blank' href='https://hub.docker.com/r/linuxserver/scrutiny'>https://hub.docker.com/r/linuxserver/scrutiny</a>, available for x86-64, arm64, and armhf</p>",
```